cscs-checks/apps/amber/amber_check.py

```diff
@@ -1,22 +1,27 @@
 import os
-
+import reframe as rfm
```

Comment:
Put an empty line here to distinguish between the standard library modules and the ReFrame ones.